coderd/database/dbtestutil/postgres_test.go

```diff
@@ -44,6 +48,9 @@ func TestOpen_InvalidDBFrom(t *testing.T) {
 
 func TestOpen_ValidDBFrom(t *testing.T) {
 	t.Parallel()
+	if !dbtestutil.WillUsePostgres() {
+		t.Skip("this test requires postgres")
+	}
```

Comment:
Flipping this generates a bit less test output in the logs
```suggestion
	if !dbtestutil.WillUsePostgres() {
		t.Skip("this test requires postgres")
	}
	t.Parallel()
```

Reply:
I changed it in newly added tests, but I noticed that across database package we use:
```go
t.Parallel()
if !dbtestutil.WillUsePostgres() {
    t.Skip("this test requires postgres")
}
```

Reply:
@Emyrk 
Turns out it fails the linter: https://github.com/coder/coder/actions/runs/13570489300/job/37933949541

`Error: coderd/database/querier_test.go:2169:6: TestGetProvisionerJobsByIDsWithQueuePosition should call t.Parallel on the top level as well as its subtests (tparallel)`

Reply:
TIL :+1: